tests/tree_inference/test_logprob.py

```diff
@@ -0,0 +1,137 @@
+"""Tests of the log-probability calculations."""
+# _logprob.py
+
+import jax.numpy as jnp
+
+import pyggdrasil.tree_inference._logprob as logprob
+from pyggdrasil.tree_inference._tree import Tree
+import pyggdrasil.tree_inference._tree as tr
+
+
+def test_compute_mutation_likelihood():
```

Comment:
Applied example as in 
[SCITE_mutation_likelihood.pdf](https://github.com/cbg-ethz/PYggdrasil/files/11289403/SCITE_mutation_likelihood.pdf)